src/FormSuccesHandlerInterface.php

```diff
@@ -1,17 +1,18 @@
 <?php
 namespace Hostnet\Component\Form;
 
-use Symfony\Component\HttpFoundation\Request;
+@trigger_error(sprintf(
+    '%s is deprecated, use the %s instead.',
```

Comment:
I'd remove 'the' here. Tiny readability improvement, since it's a full class path, instead of just a class name.